### PR TITLE
Feature/doc redirect

### PR DESCRIPTION
**Pull request by agduncan94**
* Navbar documentation link now goes to the docs site
* Will attempt to redirect existing links

[Issue #980](https://github.com/ga4gh/dockstore/issues/980)

### CODE REVIEW

**Comment by denis-yuen** [comment]
Gave this a shot, it looks like https://dockstore.org/docs/user-created is missing from http://docs.dockstore.org/docs/

File: src/app/docs/docs.component.html [hunk 1]
```diff
@@ -17,4 +17,13 @@
 <app-header>
   <img src="../assets/images/dockstore/dockstore-documentation.png"> Documentation
 </app-header>
-<router-outlet></router-outlet>
+
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="text-center">Dockstore documentation has moved to it's own site!</h2>
```
Comment (denis-yuen) [inline src/app/docs/docs.component.html]: nitpick "its"?

File: src/app/docs/docs.component.html [hunk 2]
```diff
@@ -17,4 +17,13 @@
...
+      <p class="text-center">You can find the new docs at the following link:</p>
+      <p class="text-center"><a href="{{redirectLink}}">{{getPrettyLink()}}</a></p>
```
Comment (denis-yuen) [inline src/app/docs/docs.component.html]: Maybe redirect after a few seconds https://stackoverflow.com/questions/3292038/redirect-website-after-certain-amount-of-time

**Comment by agduncan94** [comment]
Oops, where would the user-created  docs go?

**Comment by denis-yuen** [comment]
Deleted a bunch of other stuff that didn't seem needed, double-check it. 
User-created docs, not sure actually. Maybe a new section?

**Comment by denis-yuen** [comment]
(I think I'm too involved to submit github reviews now)

File: src/app/docs/docs.component.ts [hunk 3]
```diff
@@ -22,9 +22,55 @@ import { Component, OnInit } from '@angular/core';
...
+  // Generate redirect link based on path
+  getLink() {
+    const currentPath = window.location.pathname;
+    const redirectBase = 'http://docs.dockstore.org';
+    let redirectPath = '/docs';
+
+    // Iterate over each docMapping
+    for (const doc of this.docMapping) {
+      // Remove trailing /
+      const filteredPath = currentPath.replace(/\/$/, '');
```
Comment (garyluu) [inline src/app/docs/docs.component.ts]: Looks like the trailing only needs to be removed once outside the loop

File: src/app/docs/docs.component.ts [hunk 4]
```diff
@@ -22,9 +22,55 @@ import { Component, OnInit } from '@angular/core';
...
+  }
+
+  // Generate redirect link based on path
+  getLink() {
+    const currentPath = window.location.pathname;
+    const redirectBase = 'http://docs.dockstore.org';
+    let redirectPath = '/docs';
+
+    // Iterate over each docMapping
+    for (const doc of this.docMapping) {
```
Comment (garyluu) [inline src/app/docs/docs.component.ts]: Might be able to replace this loop with this.docMapping.find(...)

**Pull request merged**